tests/modules/nf-core/odgi/sort/test.yml

```diff
@@ -0,0 +1,9 @@
+- name: "odgi sort"
+  command: nextflow run ./tests/modules/nf-core/odgi/sort -entry test_odgi_sort -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/odgi/sort/nextflow.config
+  tags:
+    - "odgi"
+    - "odgi/sort"
+  files:
+    - path: "output/odgi/test.og"
+    - path: "output/odgi/versions.yml"
```

Comment:
No need to test for the `versions.yml` file
```suggestion
```

Reply:
Apparently only the md5sum check for `versions.yml` needs to be removed (see https://nfcore.slack.com/archives/CE5LG7WMB/p1666265866115999)

Reply:
OK, thanks @nvnieuwk good to know I started to delete it in my reviews from this [comment](https://github.com/nf-core/modules/pull/2328#discussion_r999065900) 😆

Reply:
No problem, I've been doing that for a while too now. That was apparently not right :grin: